src/System.Management.Automation/utils/tracing/SysLogProvider.cs

```diff
@@ -0,0 +1,504 @@
+#if UNIX
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Resources;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+
+using System.Management.Automation.Internal;
+
+namespace System.Management.Automation.Tracing
+{
+    /// <summary>
+    /// Encapsulates the message resource and SysLog logging for an ETW event.
+    /// The other half of the partial class is generated by EtwGen and contains a
+    /// static dictionary containing the event id mapped to the associated event meta data
+    /// and resource string reference.
+    /// </summary>
+    /// <remarks>
+    /// This component logs ETW trace events to syslog.
+    /// The log entries use the following common format
+    ///     (commitId:threadId:channelid) [context] payload
+    /// Where:
+    ///     commitId: A hash code of the full git commit id string
+    ///     threadid: The thread identifier of calling code.
+    ///     channelid: The identifier for the output channel. See PSChannel for values.
+    ///     context: Dependent on the type of log entry
+    ///     payload: Dependent on the type of log entry.
+    /// Note:
+    ///     commitId, threadId, and eventId are logged as HEX without a leading
+    ///     '0x'.
+    ///
+    /// 4 types of log entries are produced.
+    /// NOTE: Where constant string are logged, the template places the string in
+    /// double quotes. For example, the GitCommitId log entry uses "GitCommitId"
+    /// for the context value.
+    ///
+    /// Note that the examples illustrate the output from SysLogProvider.Log,
+    /// Data automatically prepended by syslog, such as timestamp, hostname, ident,
+    /// and processid are not shown
+    ///
+    /// GitCommitId
+    ///   This is the first log entry for a session. It provides a correlation
+    ///   between the full git commit id string and a hash code used for subsequent
+    ///   log entries.
+    ///   Context: "GitCommitId"
+    ///   Payload: string "Hash:" hashcode
```

Comment:
This description does not describe the guid in the example.

Reply:
I'll fix it.

Reply:
Fixed.

Reply:
I don't see the change.

Reply:
Fixed